modules/electrs.nix

```diff
@@ -5,11 +5,13 @@ let
   cfg = config.services.electrs;
   inherit (config) nix-bitcoin-services;
   secretsDir = config.nix-bitcoin.secretsDir;
-  index-batch-size = "${if cfg.high-memory then "" else "--index-batch-size=10"}";
-  jsonrpc-import = "${if cfg.high-memory then "" else "--jsonrpc-import"}";
 in {
   imports = [
     (mkRenamedOptionModule [ "services" "electrs" "nginxport" ] [ "services" "electrs" "TLSProxy" "port" ])
+    (mkRemovedOptionModule [ "services" "electrs" "high-memory" ] ''
+      This option has been removed because its implementation was defective, leading
```

Comment:
I have never used this module outside of testing but I don't get what youre' saying here. If `high-memory` was set, then jsonrpc-import was _not_ set. The [electrs docs](https://github.com/romanz/electrs/blob/master/doc/usage.md#usage) state that if these options are not set in a 2GB memory system, then there will be `memory allocation of xxxxxxxx bytes failedAborted`. In @nixbitcoin's case this happened with 4GB memory even.

Reply:
I completely misread that if-statement. Sorry, that's embarrassing. Reverted.